test/functional/apps/management/_test_huge_fields.js

```diff
@@ -4,13 +4,14 @@ export default function ({ getService, getPageObjects }) {
   const esArchiver = getService('esArchiver');
   const PageObjects = getPageObjects(['common', 'home', 'settings']);
 
-  describe('test large number of fields @skipcloud', function () {
+  describe.skip('test large number of fields @skipcloud', function () {
```

Comment:
pls add comment as well:
// Continuously Failed, data is too large
// Backlog: https://sim.amazon.com/issues/P50402631

Reply:
Completed